web/src/app/Routes.tsx

```diff
@@ -40,15 +37,6 @@ const WPSRoutes: React.FunctionComponent = () => {
             path={PERCENTILE_CALC_ROUTE}
             element={<PercentileCalculatorPageWithDisclaimer showDisclaimer={shouldShowDisclaimer} />}
           />
-          <Route path={FIRE_WEATHER_ROUTE} element={<Navigate to={MORECAST_ROUTE} />} />
-          <Route
-            path={MORECAST_ROUTE}
-            element={
-              <AuthWrapper>
-                <MoreCastPage />
-              </AuthWrapper>
-            }
-          />
```

Comment:
It's not in the acceptance criteria, but the description mentioned re-directing traffic to Morecast 2. The apps are so different I'm not sure that this would make sense. I wonder if anyone actually uses Morecast and if we should have a page that indicates the app has been deprecated?